cmd/datamon/cmd/cli_workshop_test.go

```diff
@@ -343,8 +346,51 @@ func TestWorkshop(t *testing.T) {
 	})
 	require.NoError(t, err)
 
-	// no test here for web (see pkg/web)
 	// no test here for upgrade (see self_upgrade_test.go)
+
+	logger.Info("test web")
+	freePort, err := GetFreePort()
+	require.NoError(t, err, "find free port")
+	port := strconv.Itoa(freePort)
+
+	cmd, pipe = testCommand(t, true, false, targetBinary,
+		"web",
+		"--port", port,
+		"--no-browser",
+	)
+
+	require.NoError(t, cmd.Start())
+	var killedWeb bool
+	defer func() {
+		if !killedWeb {
+			_ = cmd.Process.Kill()
+		}
+	}()
+
+	waiter := makeBlockingTestWaiterForReader(
+		t, "serving datamon UI", 5*time.Second,
+	)
+
+	logger.Info("waiting for web process to be ready")
+	waiter(logger, pipe)
+	logger.Info("web process ready")
+
+	resp, err := http.Get("http://localhost:" + port + "/")
+	require.NoError(t, err, "curl web service")
+	// notdo: replete test of entire http api
+	//   (contained in `pkg/web` tests)
+
+	require.NotNil(t, resp)
+
+	defer resp.Body.Close()
+	doc, err := goquery.NewDocumentFromReader(resp.Body)
+	require.NoError(t, err, "parse body html")
+
+	require.NotNil(t, doc)
+
+	logger.Info(fmt.Sprintf("home page:\n%v", doc.Text()))
+
+	// ??? more response content validation?
```

Comment:
no need to `???` here, since you already gave the answer in the body of this PR.
Suggesting instead:
```
// smoke test only: no content validation
```

Reply:
👍